src/ReactiveDomain/ReactiveDomain.Tests/Subscribers/QueuedSubscriber/TestCommandSubscriber.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System.Threading;
+using ReactiveDomain.Bus;
+using ReactiveDomain.Messaging;
+using ReactiveDomain.Tests.Helpers;
+
+namespace ReactiveDomain.Tests.Subscribers.QueuedSubscriber
+{
+    public class TestCommandSubscriber :
+                IHandleCommand<InformUserCmd>,
+                IHandleCommand<TestCommands.TestCommand3>
+    {
+        public long CommandHandled;
```

Comment:
For encapsulation, can this be a public property with a private setter?

Reply:
"A property or indexer may not be passed as an out or ref parameter"

Reply:
Depending on where it's being called from, might a property with a backing field work?